examples/ads.amp.html

```diff
@@ -920,6 +921,16 @@ <h2 id="plista">Plista responsive widget</h2>
       data-categories="politik">
   </amp-embed>
 
+  <h2 id="polymorphicads">polymorphicAds</h2>
+  <amp-ad width=320 height=50
+      type="polymorphicads"
+      data-adunit="7c0b3ae742beccf94f7726ea832277a2"
+      data-params='{"testMode": "true"}'
+      >
+    <div placeholder></div>
+    <div fallback></div>
```

Comment:
please remove `placeholder` and `fallback`. also move the `>` to the previous line.